<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LMHRecoveryForm</class>
 <widget class="QWidget" name="LMHRecoveryForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>282</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="MainLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="ptChoice" name="MaskType1"/>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="ptInput" name="Strength1"/>
   </item>
   <item>
    <widget class="ptInput" name="LowerLimit1"/>
   </item>
   <item>
    <widget class="ptInput" name="UpperLimit1"/>
   </item>
   <item>
    <widget class="ptInput" name="Softness1"/>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="ptChoice" name="MaskType2"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="ptInput" name="Strength2"/>
   </item>
   <item>
    <widget class="ptInput" name="LowerLimit2"/>
   </item>
   <item>
    <widget class="ptInput" name="UpperLimit2"/>
   </item>
   <item>
    <widget class="ptInput" name="Softness2"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ptInput</class>
   <extends>QProgressBar</extends>
   <header>ptInput.h</header>
  </customwidget>
  <customwidget>
   <class>ptChoice</class>
   <extends>QComboBox</extends>
   <header>ptChoice.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
